### PR TITLE
Expose selectedItems and remove padding on suggestion itemButton

### PR DESCRIPTION
**Pull request by amyngu**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes

- Expose selectedItems on ExtendedBasePicker
- Remove padding on suggestion itemButton, to allow consumer to have complete control of suggestion render

### CODE REVIEW

**Comment by amyngu** [comment]
@joschect can you take a look at this?

File: packages/office-ui-fabric-react/src/components/ExtendedPicker/BaseExtendedPicker.tsx [hunk 1]
```diff
@@ -87,6 +87,10 @@ export class BaseExtendedPicker<T, P extends IBaseExtendedPickerProps<T>>
     return this.input.current && this.input.current.inputElement;
   }
 
+  public get selectedItems(): T[] {
+    return this.selectedItemsList.current ? this.selectedItemsList.current.selectedItems() : [];
```
Comment (joschect) [inline packages/office-ui-fabric-react/src/components/ExtendedPicker/BaseExtendedPicker.tsx]: I strongly advise against this. Instead users should be using onchanged to track it. If we must expose it like this be aware of the potential interaction when some implementations of basepicker autoselect the first item in the suggestions list when it gets resolved.

Comment (amyngu) [reply packages/office-ui-fabric-react/src/components/ExtendedPicker/BaseExtendedPicker.tsx]: I see, I'll remove this then. Thanks!

File: packages/office-ui-fabric-react/src/components/SelectedItemsList/BaseSelectedItemsList.tsx [hunk 2]
```diff
@@ -127,6 +127,10 @@ export class BaseSelectedItemsList<T, P extends IBaseSelectedItemsListProps<T>>
     this.selection.setAllSelected(false);
   }
 
+  public selectedItems(): T[] {
+    return this.selection.getSelection() as T[];
```
Comment (joschect) [inline packages/office-ui-fabric-react/src/components/SelectedItemsList/BaseSelectedItemsList.tsx]: Do you still need this if you remove selectedItems?

**Pull request merged**